src/pages/product-details/tests/__snapshots__/product-details.spec.js.snap

```diff
@@ -6,11 +6,11 @@ exports[`Product details test Should render product details component 1`] = `
     className="undefined  "
   >
     <ProductPath />
-    <Link
-      to="/catalog/:category"
+    <WithStyles(ForwardRef(Button))
```

Comment:
I think this test should be rewritten on RTL

Reply:
Will you help your  teammate? 
Do we really have time for that?